client/src/components/shared/CompanyDetails.js

```diff
@@ -2,14 +2,10 @@
 import React from 'react'
 import CompanyDetailWrapper from '../../dataWrappers/CompanyDetailWrapper'
 import type {CompanyDetailProps} from '../../dataWrappers/CompanyDetailWrapper'
-import OldInfo from '../shared/Info/OldInfo'
 import Info from '../shared/Info/Info'
 
-const CompanyDetails = ({company, oldCompany, useNewApi}: CompanyDetailProps) =>
-  useNewApi ? (
-    <Info data={company} />
-  ) : (
-    <OldInfo data={oldCompany} eid={oldCompany.eid} />
-  )
+const CompanyDetails = ({company, oldCompany, useNewApi}: CompanyDetailProps) => (
```

Comment:
Pls remove also `oldCompany` prop.

Reply:
And `useNewApi`. And also on all places, where it's included :slightly_smiling_face: